Package.swift

```diff
@@ -1,23 +1,28 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.7
```

Comment:
Just saying, Swift 5.8 is out :)

Reply:
Yeah, I probably wouldn't have any issues with it here, but I just encountered problems with 5.8 in another project so I'm staying on 5.7 for now 😅